### PR TITLE
Tools GUI windows - disable click through UI background

### LINKED ISSUE #749: Panel backgrounds aren't absorbing mouse interactions
### Describe the problem

When you click on an option, like Stoplight manager, or speedlimit, any of them, the new black window pops up over the other buttons, and when you click on something, it also clicks on the menu buttons underneath it. 

### Log files

[TMPE.log](https://github.com/CitiesSkylinesMods/TMPE/files/4263082/TMPE.log)
[output_log.txt](https://github.com/CitiesSkylinesMods/TMPE/files/4263083/output_log.txt)

### Screenshots?

parking restrictions
nodes

### LINKED ISSUE #934: Moving main panel in the background
If opened speed limit panel and main panel is under it, then if drag speed panel, main panel move too

### PR DESCRIPTION
**Pull request by krzychu124**
Closes #934 
Closes #749 

Additional changes:
Hotfix for `What's New` panel version check - throws `NullReferenceException` if working with default values in GlobalConfig (can compare `Version` with `null` but not the other way around)

### CODE REVIEW

**Comment by aubergine10** [comment]
Code looks good, will test in game tonight.

**Review by aubergine10 — APPROVED** [review]
Works as advertised as far as I can tell. 👍 

Wasn't sure what the secondary window was, but assuming it works on same principle it should work as expected.

File: TLM/TLM/UI/WhatsNew/WhatsNew.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class WhatsNew {
             }
         }
 
-        public bool Shown => PreviouslySeenVersion >= CurrentVersion;
+        public bool Shown => CurrentVersion <= PreviouslySeenVersion;
```
Comment (aubergine10) [inline TLM/TLM/UI/WhatsNew/WhatsNew.cs:34]: Maybe add code comment here explaining prev version can be null?

**Review by chameleon-tbn — APPROVED** [review]
Lgtm

**Pull request merged**